docs/guides/ipfs.md

```diff
@@ -0,0 +1,110 @@
+# IPFS
+
+IPFS (InterPlanetary File System) is a suite of protocols for publishing data (files, directories, websites, etc.) in a decentralised fashion.
+For more info, see [What is IPFS](https://docs.ipfs.tech/concepts/what-is-ipfs/).
+
+There is an option to use some Lido interfaces via IPFS, for example Lido Ethereum Staking Widget.
+
+#### We use IPFS because:
+- IPFS has no single point of failure. The failure of a single or even multiple nodes in the network does not affect the functioning of the entire network.
+- IPFS is decentralized, which makes IPFS more resilient than traditional systems.
+- IPFS uses cryptographic hashes to verify the authenticity and integrity of files, making it difficult for malicious actors to affect files.
+
+## Address
+### What is a CID
+A content identifier, or CID, is a label used to point to material in IPFS. CIDs are based on the content’s cryptographic hash.
+Any difference in the content will produce a different CID.
+But CIDs won't match file hashes (checksums), because CID contains additional information that the hash does not.
+
+### IPFS HTTP Gateways
+An IPFS gateway is a web-based service that gets content from an IPFS network, and makes it available via HTTP protocol 
+that all web browsers understand.  
+A gateway address can look like this: `https://{CID}.ipfs.cf-ipfs.com`
+
+:::note
+Some browsers support IPFS natively and can access IPFS content without gateways, using canonical addressing like  
+`ipfs://{CID}/{optional path to resource}`  
+
+Also, there is the [IPFS Companion Browser Extension](https://docs.ipfs.tech/install/ipfs-companion), which enables support for IPFS in your browser. 
+:::
+
+### Where to get CID and gateway address
+
+:::info
+Each new set of changes to an application will produce a new CID, therefore each release will be available at its specific address.
+This means **there won't be a gateway address that always points to the most recent release**.
+A gateway you are using now may point to the most updated version, but until a new release happens.  
+So, you may want to look for a new CID or gateway address from time to time.
+However, you may continue to use this gateway until it stops working for a some reason.   
```

Comment:
This is also not entirely true.
It may not be safe to continue using the old version on the old CID, although it will work.
I think this can just be removed

Reply:
By these words I wanted to say that it is technically possible, didn't want to say, that it is optimal.
I'll just remove it, okay